docs/csharp/language-reference/operators/conditional-operator.md

```diff
@@ -10,70 +10,76 @@ helpviewer_keywords:
   - "conditional operator (?:) [C#]"
 ms.assetid: e83a17f1-7500-48ba-8bee-2fbc4c847af4
 ---
-
 # ?: Operator (C# Reference)
 
-The conditional operator (`?:`), commonly known as the ternary conditional operator, returns one of two values depending on the value of a Boolean expression. Following is the syntax for the conditional operator.  
+The conditional operator `?:`, commonly known as the ternary conditional operator, evaluates one of two expressions and returns its result depending on the value of a Boolean expression. Beginning with C# 7.2, the [conditional ref expression](#conditional-ref-expression) returns the reference to the result of one of the two expressions.
+
+The syntax for the conditional operator is as follows:
 
 ```csharp
-condition ? first_expression : second_expression;  
+condition ? consequence : alternative
 ```
 
-Beginning with C# 7.2, the `first_expression` and `second_expression` my be [`ref` expressions](https://github.com/dotnet/csharplang/blob/master/proposals/csharp-7.2/conditional-ref.md):
+The `condition` expression must evaluate to `true` or `false`. If `condition` evaluates to `true`, the `consequence` expression is evaluated and its result becomes the result of the operation. If `condition` evaluates to `false`, the `alternative` expression is evaluated and its result becomes the result of the operation. A conditional expression evaluates only one of the two expressions: either `consequence` or `alternative`.
```

Comment:
- comma after both occurrences of "is evaluated"
- The last sentence can be read as a repetition of the previous sentences in the paragraph. Perhaps just "Only \`consequence\` or \`alternative` is evaluated.